.github/workflows/publish-pypi.yaml

```
@@ -2,11 +2,9 @@
 name: Publish Python 🐍 distribution 📦 to PyPI
 
 on:
-  push:
+  workflow-dispatch:
```

actionlint (1.7.11)

[error] unknown Webhook event "workflow-dispatch". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)